src/test/java/com/iexec/worker/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.iexec.worker.utils;
+
+import java.lang.reflect.Field;
+
+public class ReflectionUtils {
```

Comment:
Why did you create a new class instead of using existing Spring one ?
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/ReflectionUtils.html

Reply:
The idea was to have a single method to:
- get the `Field` instance;
- make it accessible;
- get the value of the field.

Spring's `ReflectionUtils` class does not provide this type of method. However, we can refactor my implementation using Spring, which would be easier to read:

```java
    public static <R, T> R getFieldAndSetAccessible(T object, String fieldName) {
        final Field field = org.springframework.util.ReflectionUtils.findField(object.getClass(), fieldName);
        org.springframework.util.ReflectionUtils.makeAccessible(field);
        return (R) org.springframework.util.ReflectionUtils.getField(field, object);
    }
```
Do you think this would be good enough?

Reply:
hmm, I missed my first comment, there is another class dedicated to tests, `ReflectionTestUtils`.
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/util/ReflectionTestUtils.html#getField-java.lang.Object-java.lang.Class-java.lang.String-

The `getField` method seems to work around the accessibility point.

Reply:
You're right! I didn't know such a method exists! Here's a new commit: https://github.com/iExecBlockchainComputing/iexec-worker/pull/471/commits/995764034a23409a6ec05a9ab6107ae2a01e2986 :)